pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-common/src/main/java/org/apache/pinot/plugin/ingestion/batch/common/SegmentGenerationTaskRunner.java

```diff
@@ -65,6 +65,7 @@ public class SegmentGenerationTaskRunner implements Serializable {
   public static final String DEPRECATED_USE_LOCAL_DIRECTORY_SEQUENCE_ID = "local.directory.sequence.id";
   public static final String USE_GLOBAL_DIRECTORY_SEQUENCE_ID = "use.global.directory.sequence.id";
   public static final String APPEND_UUID_TO_SEGMENT_NAME = "append.uuid.to.segment.name";
+  public static final String OMIT_TIMESTAMPS_IN_SEGMENT_NAME = "omit.timestamps.in.segment.name";
```

Comment:
The change looks good otherwise. 
Can we rename to "**append.timestamps.to.segment.name**", to be consistent with how its done for uuid (append.uuid.to.segment.name). Also we creating the name here, so might be intuitive to  include/not include?

Reply:
with `append.timestamps.to.segment.name`, we'd have to make the value `true` by default to match existing behavior. it seems a little unintuitive to default a parameter to true rather than false here.

> so might be intuitive to include/not include?

sorry i'm not following this comment. what do you mean to include?

Reply:
Im fine with either approach. I just thought having the same naming convention would make it easier to use this class, but I guess this would matter only to users who want to override defaults.

Can we just rename this to **exclude.timestamps.in.segment.name** ?  Looks good other wise. Thanks !